From 86cef04bac1635f0d8a16be943980e6338720b53 Mon Sep 17 00:00:00 2001
From: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@linux.intel.com>
Date: Wed, 18 Nov 2020 11:25:49 +0000
Subject: [PATCH 0246/1418] x86/tdx: Add config options for platform fixes

Add new config options to selectively turn on chip specific
fixes.

Signed-off-by: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@linux.intel.com>
---
 arch/x86/Kconfig | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 4a7b68797f53..ad0b7520860b 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -897,6 +897,14 @@ config TDX_FUZZ
 	help
 	  Support fuzzing of TDX guests against attacks from hosts
 
+config INTEL_TDX_ICL_FIXES
+	bool "Fixes specific for ICL chip"
+	depends on INTEL_TDX_GUEST
+
+config INTEL_TDX_FIXES
+	bool "Platform specific fixes"
+	depends on INTEL_TDX_GUEST
+
 endif #HYPERVISOR_GUEST
 
 source "arch/x86/Kconfig.cpu"
-- 
2.31.1

